books/free-programming-books-fr.md

```diff
@@ -148,8 +148,8 @@
 
 ### Git
 
-* [Git Magic](http://www-cs-students.stanford.edu/~blynn/gitmagic/intl/fr/) - Alexandre Garel, Paul Gaborit, Nicolas Deram
-* [Pro Git](http://www.git-scm.com/book/fr/v2) - Scott Chacon, Ben Straub
+* [Git Magic](http://www-cs-students.stanford.edu/~blynn/gitmagic/intl/fr/) - Ben Lynn, `trl.:` Alexandre Garel, `trl.:` Paul Gaborit, `trl.:` Nicolas Deram (online, PDF)
+* [Pro Git](http://www.git-scm.com/book/fr/) - Scott Chacon, Ben Straub (online, PDF, EPUB)
```

Comment:
```suggestion
* [Pro Git](http://www.git-scm.com/book/fr/) - Scott Chacon, Ben Straub (HTML, PDF, EPUB)
```

I think it's more appropriate

Reply:
I propose to replace all "online" with "html"

Reply:
Done at e4460f961f4a9095727df5c0813d0a97a8819cb7

Reply:
I also mean resources that are not "Pro Git", like:
https://github.com/EbookFoundation/free-programming-books/blob/570289b44b2cfff005b7a77ed5e31441015a1234/books/free-programming-books-fr.md#L151

Reply:
Check again if now is all right 4ec5e557cf9035840104b55433316649d425cc4e

Reply:
It's good but I can't resolve the conversation